commit/core.pyx

```diff
@@ -676,13 +676,24 @@ cdef class Evaluation :
         print '   [ %s ]' % ( time.strftime("%Hh %Mm %Ss", time.gmtime(self.CONFIG['optimization']['fit_time']) ) )
 
 
-    def save_results( self, path_suffix = None ) :
+    def save_results( self, path_suffix = None, save_pickle = True, save_txt = False ) :
```

Comment:
Can we rename the `save_pickle` and `save_txt` to something more intuitive? For instance, `save_opt_details` and `save_coeff`?

Reply:
I changed them with the ones you just proposed.